students/shibata/session3/mailroom.py

```diff
@@ -0,0 +1,150 @@
+#! /usr/bin/env Python
+"""
+It should have a data structure that holds a list of your donors and a history of the amounts they have donated. This structure should be populated at first with at least five donors, with between 1 and 3 donations each.
+
+You can store that data structure in the global namespace.
+The script should prompt the user (you) to choose from a menu of 3 actions: “Send a Thank You”, “Create a Report” or “quit”)
+"""
+
+from time import sleep
+import pprint
+
+if __name__ == '__main__':
+    objFileName = "C:\Files\donate.txt"
+    strData = ""
+    dicRow = {}
+    lstTable = []
+
+# Step 1
+# When the program starts, load the any data you have
+# in a text file called ToDo.txt into a python Dictionary.
+objFile = open(objFileName, "r")
+for line in objFile:
+    strData = line.split(",") # readline() reads a line of the data into 2 elements
+    dicRow = {"Name":strData[0].strip(), "Total":strData[1].strip(), "Num":strData[2].strip()}
+    lstTable.append(dicRow)
+objFile.close()
```

Comment:
better to use the "context manager"

with open(objFileName, "r") as objFile:
    ...

then it will call close for you at the end of the block.